recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -30,6 +30,9 @@ GPIO_IR_TX ?= "17"
 
 CAN_OSCILLATOR ?= "16000000"
 
+WM8960="${@bb.utils.contains("MACHINE_FEATURES", "wm8960", "1", "0", d)}"
```

Comment:
This deviates from the rest of the dtbo enablement in this recipe. None use machine features. And strictly, I'm not sure if a hat is a machine feature. I'd keep it for now as a separate variable and maybe unify it in the future somehow.

Also, we need to update the docs for this support (see existing docs).

Reply:
I'll update the docs.

Reply:
@agherzan Why is WM8960 any different than PITFT?

Reply:
I see. I've forgotten we do that for PITFT too. OK. I'm fine with the machine feature as well.